apps/web/src/components/workflow/preview/push/Content.tsx

```diff
@@ -0,0 +1,185 @@
+       */
+      const combinedContent = `${previewData.current.title} ${previewData.current.content}`;
+      getLocalesFromContent({
+        content: combinedContent,
+      });
+    }
+  }, [getLocalesFromContent, previewData]);
+
+  useEffect(() => {
+    setSelectedLocale(organization?.defaultLocale);
+  }, [organization?.defaultLocale]);
+
+  const onLocaleChange = (locale: string) => {
+    setSelectedLocale(locale);
+  };
+
+  const handleMouseEnter = () => {
+    setIsEditOverlayVisible(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsEditOverlayVisible(false);
+  };
+
+  return (
+    <ContentWrapperStyled>
+      <Group>
+        <LocaleSelect
+          isLoading={isLocalesLoading}
+          locales={locales || []}
+          onLocaleChange={onLocaleChange}
+          value={selectedLocale}
+        />
+      </Group>
+      <ContentAndOVerlayWrapperStyled
+        isError={!!errorMsg}
+        onMouseEnter={handleMouseEnter}
+        onMouseLeave={handleMouseLeave}
+      >
+        {isEditOverlayVisible && <PreviewEditOverlay />}
+        <ContentStyled isBlur={isEditOverlayVisible}>
+          {isLoading ? (
+            <HeaderSkeleton />
+          ) : (
+            <ContentHeaderStyled>
+              <Flex align="center" gap={5}>
+                <NovuGreyIcon color={isDark ? colors.B30 : '#1F1F27'} width="24px" height="24px" />
+                <Text color={colors.B20} weight="bold">
+                  Your App
+                </Text>
+              </Flex>
+              <Text color={colors.B60}>now</Text>
+            </ContentHeaderStyled>
+          )}
+          {isLoading ? (
+            <ContentSkeleton />
+          ) : (
+            <div>
+              <Text color={colors.B15} weight="bold" rows={1}>
+                {parsedPreviewState.title || ''}
+              </Text>
+              <Text color={colors.B15} rows={3}>
+                {parsedPreviewState.content || ''}
+              </Text>
+            </div>
+          )}
+        </ContentStyled>
+      </ContentAndOVerlayWrapperStyled>
+
+      {errorMsg && !isLoading && <Text color={colors.error}>{errorMsg}</Text>}
+    </ContentWrapperStyled>
+  );
+}
+
+const HeaderSkeleton = () => {
+  return (
+    <Group position="apart" noWrap>
+      <Group spacing={16} noWrap>
+        <SkeletonStyled radius={6} height={24} width={24} />
+        <SkeletonStyled height={14} width={80} radius={6} />
+      </Group>
+      <SkeletonStyled height={14} width={40} radius={6} />
+    </Group>
+  );
+};
+
+const ContentSkeleton = () => {
+  return (
+    <Stack spacing={10} w="100%">
+      <SkeletonStyled height={14} width="70%" radius={6} />
+      <SkeletonStyled height={14} width="35%" radius={6} />
+    </Stack>
+  );
+};
+
+const SkeletonStyled = styled(Skeleton)`
+  &::before {
+    background: rgba(0, 0, 0, 0.08);
+  }
+
+  &::after {
+    background: rgba(0, 0, 0, 0.08);
+  }
+`;
```

Comment:
suggestion: group these two into one skeleton component and show when `isLoading`